src/components/OntologyBrowser/Test/SelectionTree.test.tsx

```diff
@@ -0,0 +1,145 @@
+import { TreeNode } from '../store';
+import { SelectionTree } from '../SelectionTree';
+import * as React from 'react';
+import Enzyme, {shallow, mount} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { treeData } from './mockData';
+
+Enzyme.configure({adapter: new Adapter()});
+
+describe('In SelectionTree', () => {
+  const checkedKeys: string[] = [];
+  const targetKeys: string[] = [];
+  const onItemSelect = jest.fn();
+  const dataSource: TreeNode[] = treeData
+
+  beforeAll(() => {
+  });
+
+  it('should renders without crashing', () => {
```

Comment:
'should renders selection tree' better to have positive structure.

Also, you dont expect anything here, could fail silently. It need at least one expect